lib/src/signed_video_internal.h

```diff
@@ -208,15 +209,19 @@ struct _gop_info_t {
   uint8_t *gop_hash;  // Pointing to the memory slot of the gop_hash in |hashes|.
   uint8_t hash_list[HASH_LIST_SIZE];  // Pointer to the list of hashes used for
   // SV_AUTHENTICITY_LEVEL_FRAME.
+  uint8_t nalu_hash_list[HASH_LIST_SIZE];
```

Comment:
Is this an intermediate member while implementing the "new gop hash" feature or necessary for the future?
If it is temporary, I suggest you move it to the bottom of the struct and add a TODO that it will be removed once the feature is fully implemented. Otherwise, move it to after `list_idx` and add a description and explain why there is need for a second hash list.